Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -2961,7 +2961,7 @@ public class CheckHasNoAssumes_Visitor : BottomUpVisitor {
       }
     }
 
-    protected internal void TrStmt(Statement stmt, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts = null) {
+    protected void TrStmt(Statement stmt, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts = null) {
```

Comment:
Do we need that?

Reply:
`protected internal` exposes more than `protected`, and `internal` wasn't needed here. It's suggested by Rider.